docs/core/entity/time.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Time Entity
+sidebar_label: Time
+---
+
+A `time` is an entity that allows the user to input a time to an integration. Derive entity platforms from [`homeassistant.components.time.TimeEntity`](https://github.com/home-assistant/core/blob/dev/homeassistant/components/time/__init__.py)
+
+## Properties
+
+:::tip
+Properties should always only return information from memory and not do I/O (like network requests). Implement `update()` or `async_update()` to fetch data.
+:::
+
+| Name | Type | Default | Description
+| ---- | ---- | ------- | -----------
+| native_value | time | **Required** | The value of the time.
+
+Other properties that are common to all entities such as `icon`, `name` etc are also applicable.
```

Comment:
```suggestion
Other properties that are common to all entities such as `icon`, `name` etc. are also applicable.
```
Maybe consider adding a link to where these common properties are documented?

Reply:
Pulled this statement from another entity doc. If we want to centralize this information we should do a sweep of all entity docs in a separate PR.

Reply:
Fair enough 👍